webpack/scenes/Subscriptions/Manifest/SimpleContentAccess.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Switch } from '@patternfly/react-core';
+import { OverlayTrigger, Tooltip, Icon } from 'patternfly-react';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+function SimpleContentAccess(props) {
+  const {
+    canToggleSimpleContentAccess,
+    simpleContentAccessValue,
+    enableSimpleContentAccess,
+    disableSimpleContentAccess,
+    setActionInProgress,
+  } = props;
+
+  const [isChecked, setChecked] = useState(simpleContentAccessValue);
+
+  useEffect(() => {
+    setChecked(simpleContentAccessValue);
+  });
+
+  function toggleSimpleContentAccess() {
+    if (simpleContentAccessValue) {
+      disableSimpleContentAccess();
+    } else {
+      enableSimpleContentAccess();
+    }
+  }
+
+  function handleChange() {
+    setChecked(!isChecked);
+    setActionInProgress();
+    toggleSimpleContentAccess();
+    props.hideModal();
+  }
+
+  return (
+    <Fragment>
+      {simpleContentAccessValue ? __('Yes') : __('No')}
+      <OverlayTrigger
+        overlay={
+          <Tooltip id="sca-tooltip">
+            {__('When Simple Content Access is enabled, hosts can consume from all repositories in their Content View regardless of subscription status.')}
+          </Tooltip>
+        }
+        placement="bottom"
+        trigger={['hover', 'focus']}
+        rootClose={false}
+      >
+        <Icon type="pf" name="info" />
+      </OverlayTrigger>
+      <span id="manifest-toggle-sca-switch">
+        <Switch
+          id="simple-switch"
+          isChecked={isChecked}
+          onChange={handleChange}
+          isDisabled={!canToggleSimpleContentAccess}
+        />
+        <OverlayTrigger
+          overlay={
+            <Tooltip id="sca-tooltip">
+              {__('Toggling Simple Content Access will refresh your manifest.')}
```

Comment:
![sca_toggle](https://user-images.githubusercontent.com/22042343/86970449-6a5f0780-c13d-11ea-9bbe-c1211955f9da.png)

Having two info icons sandwiching the switch just looks too funny to me.  For the `Toggling Simple Content Access will refresh your manifest.`, I was envisioning regular text that's on the screen all the time, below the switch and above the Choose File button.

Also, to me the remaining info icon (line 42 above) makes more sense next to the text "Simple Content Access," not next to the switch.